components/bl-audio-player-component/src/index.js

```diff
@@ -58,11 +54,7 @@ export default function AudioPlayer({ component }) {
   component.playAudio = () => setIsPlaying(true);
   component.stopAudio = () => stopPlaying();
   component.replaceAudio = (audioUrl, audioTitle) => {
-    setCurrentTrack({
-      url  : audioUrl,
-      title: audioTitle,
-    });
-
+    setCurrentTrack({ url: audioUrl, title: audioTitle });
```

Comment:
You could rename function argument to achieve

`setCurrentTrack({ url, title });`